packages/block-editor/src/components/date-format-picker/style.scss

```diff
@@ -1,18 +1,16 @@
 .block-editor-date-format-picker {
```

Comment:
We can add CSS to make it the same as trunk, but there is a 20px space so we'd either have to use 16px, 24px, or possibly a grid unit subtracted by another in order to get 20px if we're okay to not use a single grid unit.